src/test/func/malloc/malloc.cc

```diff
@@ -0,0 +1,118 @@
+#include <cerrno>
+#include <malloc.h>
```

Comment:
This file has been deprecated for about 20 years and causes a hard error on FreeBSD, it probably should be stdlib.h.